app/Http/Controllers/Admin/ColumnMonsterCrudController.php

```diff
@@ -61,13 +61,26 @@ public function setupListOperation()
             }
         }
 
+        $miscellaneousColumns = static::getFieldsArrayForMiscellaneousTab();
+        // Adding extra attributes in "range" column definition
+        if ($miscellaneousColumns) {
+            foreach ($miscellaneousColumns as $columnKey => $miscellaneousColumn) {
+                if (isset($miscellaneousColumn['type']) && ($miscellaneousColumn['type'] == 'range')) {
+                    // Creating new variable array to over-ride date_range column as that is "unset" above
+                    $miscColumnRange = ['name' => $miscellaneousColumn['name'], 'label' => $miscellaneousColumn['label'], 'type' => $miscellaneousColumn['type'], 'progress_class' => 'progress-bar bg-success', 'is_striped' => '1', 'attributes' => $miscellaneousColumn['attributes'], 'tab' => $miscellaneousColumn['tab'], 'wrapperAttributes' => $miscellaneousColumn['wrapperAttributes']];
```

Comment:
🚩🚩🚩 Why is this line of code so long @munjaldevelopment ? Impossible to read this way, please split it up in multiple lines and properly indent it. Otherwise there's no way to understand what's happening here.

Reply:
Ok. Noted. I put array inline to understand. I will change that to multi-line again..

Reply:
I have changed array definition in better way now.

Reply:
Please check & revert.